### PR TITLE
event_manager_profiler_tracer: Rename module event_manager_profiler

### PR DESCRIPTION
**Pull request by zycz**
Rename module event_manager_profiler to event_manager_profiler_tracer

Jira: NCSDK-12480

Signed-off-by: Jan Zyczkowski <jan.zyczkowski@nordicsemi.no>

### CODE REVIEW

**Review by coderbyheart — APPROVED** [review]
**nitpick:** Explain in the commit message why the rename was neccessary.

**Comment by zycz** [comment]
> **nitpick:** Explain in the commit message why the rename was neccessary.

Hi, the rename was made because the older name could be somehow misleading.

**Review by greg-fer — APPROVED** [review]

**Review by simensrostad — APPROVED** [review]

File: samples/event_manager_profiler/prj.conf [hunk 1]
```diff
@@ -19,5 +19,5 @@ CONFIG_HEAP_MEM_POOL_SIZE=2048
 CONFIG_REBOOT=y
 
 # Configuration required by Event Manager profiler tracer
```
Comment (MarekPieta) [inline samples/event_manager_profiler/prj.conf:21]: Should we also rename the sample (for consistency) ?

Comment (MarekPieta) [reply samples/event_manager_profiler/prj.conf:21]: The test scenario name was renamed, but name of the sample was left unchanged.

**Review by MarekPieta — APPROVED** [review]

File: samples/event_manager_profiler/prj.conf (hunk 1 above)
Comment (zycz) [reply samples/event_manager_profiler/prj.conf:21]: Good point made changes

File: samples/event_manager_profiler_tracer/README.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.. _event_manager_profiling_sample:
+.. _event_manager_profiling_tracer_sample:
```
Comment (greg-fer) [inline samples/event_manager_profiler_tracer/README.rst:1]: Links using this tag need to be updated.

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Review by pdunaj — APPROVED** [review]

**Comment by pdunaj** [comment]
Hi @shantha-14 , this change is required in 1.8 for sources consistency.

**Pull request merged**